bayesian_torch/layers/flipout_layers/conv_flipout.py

```diff
@@ -631,16 +648,18 @@ def forward(self, x, return_kl=True):
 
         delta_kernel = (sigma_weight * eps_kernel)
 
-        kl = self.kl_div(self.mu_kernel, sigma_weight, self.prior_weight_mu,
-                         self.prior_weight_sigma)
+        if return_kl:
```

Comment:
Hey, I think only computing the kl value if return_kl is set to True might break things if we are plugging bayesian layers on a deterministic model. 

Maybe the best way could be if `self.dnn_to_bnn_flag` is set to True, we set it to null, but not computing it at all might just break everything - and we don't have unit tests yet to ensure things won't break.